modules/tinymce/src/core/main/ts/selection/SelectionUtils.ts

```diff
@@ -135,10 +135,16 @@ const preserve = (selection: EditorSelection, fillBookmark: boolean, executor: (
   selection.moveToBookmark(bookmark);
 };
 
+const isSelectionOverWholeNode = (range: Range, nodeTypePredicate: (n: Node) => boolean): boolean =>
```

Comment:
Could we add a test for this I know that the SelectionUitlsTest.ts file doesn't cover all the functions we have today but maybe we should try to add more tests to it as we go. Do a bit of extra work to pay of some testing coverage dept.